unobstructed_lanechange/src/unobstructed_lanechange.cpp

```diff
@@ -59,6 +59,7 @@ namespace unobstructed_lanechange
         pnh_->param<double>("curvature_calc_lookahead_count", curvature_calc_lookahead_count_, 1);
         pnh_->param<int>("downsample_ratio", downsample_ratio_, 8);
         pnh_->param<bool>("enable_object_avoidance_lc", enable_object_avoidance_lc_, false);
+        pnh_->param<double>("acceptable_time_difference_", acceptable_time_difference_, 1.0);
```

Comment:
I think you need to have a `time_dur_ = ros::Duration(acceptable_time_difference_)` here as well, otherwise this just updates that double value member but the ros::Duration won't reflect that change.